### PR TITLE
chore(idf): possible renaming in idf

### PR DESCRIPTION
**Pull request by hfudev**

### CODE REVIEW

File: pytest-embedded-idf/pytest_embedded_idf/app.py [hunk 1]
```diff
@@ -52,7 +52,8 @@ def __init__(
         self.elf_file = self._get_elf_file()
 
         # loadable elf file skip the rest of these
-        if self.sdkconfig.get('APP_BUILD_TYPE_ELF_RAM'):
+        # TODO to be improved in #186
+        if self.sdkconfig.get('APP_BUILD_TYPE_ELF_RAM') or self.sdkconfig.get('APP_BUILD_TYPE_RAM'):
```
Comment (igrr) [inline pytest-embedded-idf/pytest_embedded_idf/app.py:56]: As an alternative, can we set app.is_loadable_elf via pytest.mark.parametrize in the test case itself?

Comment (hfudev) [reply pytest-embedded-idf/pytest_embedded_idf/app.py:56]: IMHO, I prefer #186 instead of setting it explicitly, since users may forget to set it again.

Comment (igrr) [reply pytest-embedded-idf/pytest_embedded_idf/app.py:56]: Actually, what is the point of #186? `sdkconfig` file already contains all the options added by sdkconfig.rename, so if `APP_BUILD_TYPE_ELF_RAM` got renamed into `APP_BUILD_TYPE_ELF`, then `sdkconfig` should contain both options.

**Review by igrr — APPROVED** [review]

File: pytest-embedded-idf/pytest_embedded_idf/app.py (hunk 1 above)
Comment (hfudev) [reply pytest-embedded-idf/pytest_embedded_idf/app.py:56]: yes. sorry I didn't explain #186 clearly. The reason that we're having this issue is like you said, the sdkconfig.json doesn't contain the history names.

**Pull request merged**